nn-specials/src/main/java/org/frankframework/extensions/rekenbox/Adios2XmlPipe.java

```diff
@@ -384,7 +384,7 @@ public String makeXml(String s, PipeLineSession session) throws PipeRunException
 				bericht.addSubElement(rubriek);
 			}
 		}
-		return bericht.toXML(true);
```

Comment:
Deze was vroeger true, deed xmlHeader niets?

Reply:
Daar werd inderdaad niets mee gedaan in de toXml